i18n-util/yarn.lock

```diff
@@ -0,0 +1,4797 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
+"@babel/code-frame@^7.0.0", "@babel/code-frame@^7.10.4":
+  version "7.12.11"
+  resolved "https://artifacts.aue1d.saasure.com:443/artifactory/api/npm/npm-okta-master/@babel/code-frame/-/code-frame-7.12.11.tgz#f4ad435aa263db935b8f10f2c552d23fb716a63f"
```

Comment:
If we expect the doc team to build/run this - we'll need to move off of the internal artifactory and use public NPM for these resolved dependencies.

Reply:
Good point. Although I am not sure if the doc team is going to build/run this package.

Reply:
We might be able to get them VPN access if they do.